### PR TITLE
update for lead form tracking

### PR DESCRIPTION
**Pull request by jasdeepg**

### CODE REVIEW

**Comment by jasdeepg** [comment]
@bpapillon tried to test this using https://github.com/RootsRated/support-documentation-template, but I'm still not certain if I'm doing so in the right way. I can't seem to get the site to render locally nor update from support-documentation.

File: Services/curl https:::api.hubapi.com:deals:v1:deal:paged?ha [hunk 1]
```diff
@@ -0,0 +1,4 @@
+curl https://api.hubapi.com/deals/v1/deal/paged?hapikey=1e7f5076-6d55-492b-a545-1f670b52f80f&properties=dealstage&propertiesWithHistory=dealstage&includePropertyVersions=true
```
Comment (tyrp) [inline Services/curl https:::api.hubapi.com:deals:v1:deal:paged?ha]: Was this meant to be committed?

**Comment by tyrp** [comment]
This branch was not forked from the latest master, I have rebased and pushed up an update.

File: Services/curl https:::api.hubapi.com:deals:v1:deal:paged?ha (hunk 1 above)
Comment (jasdeepg) [reply Services/curl https:::api.hubapi.com:deals:v1:deal:paged?ha]: no, not sure where that came from

**Comment by jasdeepg** [comment]
@tyrp i figured i may have done that incorrectly. sorry!

**Pull request merged**